asab/metrics/openmetric.py

```diff
@@ -1,8 +1,45 @@
 import re
-import logging
 
 
-L = logging.getLogger(__name__)
+def metric_to_text(metric_record):
+	if metric_record.get("Values") is None:
+		return
+	metric_lines = []
+	# TODO: resetable counter is gauge - but how do I recognize counter that is not resetable?
+	if metric_record.get("Type") == "Histogram":
```

Comment:
@eliska-n This is weird even on my standards. :D Here you limit the types to only "histogram" and "gauge". It should be unified somewhere, because when people add a new type and they do not go to this method, it will simply treat all other types as gauge. There should be a list of available types somewhere and used here. In case, there is a missing implementation for the given type, it should cause RuntimeError etc.

Reply:
Yes. Making all other metrics Gauges is too simplified. However, from prometheus point of view: resetable counter is gauge. dutycycle is gauge... I need to find a way how to recognize not reseted metrics.